openedx/features/announcements/static/announcements/jsx/announcements.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+class Announcement extends React.Component {
```

Comment:
This code needs to be tested too.  See https://github.com/edx/paragon/blob/master/src/StatusAlert/StatusAlert.test.jsx for example.

Reply:
JSX tests still need to be added?